data/src/test/java/org/apache/iceberg/data/TestLocalScan.java

```diff
@@ -449,6 +454,45 @@ private DataFile writeFile(String location, String filename, List<Record> record
     }
   }
 
+  @Test
+  public void testFilterWithComplexType() throws IOException {
+    if (format == FileFormat.PARQUET) {
```

Comment:
This is due to https://issues.apache.org/jira/browse/PARQUET-1851.

Reply:
Tests should be disabled using `Assume`, so it doesn't look like they are passing and are skipped instead.

I think it makes sense to only disable the parts of the test that don't pass in Parquet -- I think just the timestamp without zone type. Can you separate that into a different case and add the assume to that one?

Reply:
This is fixed by #891.